### DISCUSSION TITLE
Feature: Do Not Disturb mode on maps/heist/etc

### DISCUSSION THREAD

**Discussion by idonotknowwhoiam** [Lailloken/Exile-UI#233] [Ideas & Feedback]
Is it possible to implement an automatic DnD mod when you are on maps/heist/etc? Sometimes you can get distracted by messages and die, so I think it could be useful.

**Comment by Lailloken** [comment]
No, it's against TOS to have automated inputs/commands based on screen-reading or client.txt logging
